docs/about-core-concepts/products.rst

```diff
@@ -185,4 +185,6 @@ Example 3D product definition for GEDI L2B cover_z:
 .. literalinclude:: ../config_samples/dataset_types/gedi_l2b_cover_z.yaml
    :language: yaml
 
-/
+.. note::
+
+    `extra_dimensions` is defined the yaml
```

Comment:
This note doesn't make any sense to me. What is it about?

Reply:
according to line 157, ```extra_dimensions (required for 3D datasets)```, and it seems this example from file ```    ../config_samples/dataset_types/gedi_l2b_cover_z.yaml``` is trying to showcase that, so I thought it could be helpful to highlight it

Reply:
Good point, but I think it could be worded better.  E.g.: "is defined IN the yaml"

Reply:
Oh okay, cool. Lets merge this, and I'll do another edit separately. :)

Reply:
lol, now i read it again, I see whats wrong